src/main/java/mission1/racing/RacingGame.java

```diff
@@ -9,27 +9,44 @@
 
 public class RacingGame {
     private int time;
-    private int maxPosition = 0;
 
     private Car[] cars;
     private List<String> winners = new ArrayList<>();
-    private RacingGameInputView inputView = new RacingGameInputView();
-    private RacingGameOutputView outputView = new RacingGameOutputView();
 
     public void announceWinners() {
         setGame();
 
         for (int i = 0; i < time; i++) startGame();
```

Comment:
자바 컨벤션을 지키지 않으셨네요. 수행문이 한줄이라도 괄호로 감싸줘야 합니다.
```suggestion
        for (int i = 0; i < time; i++) {
            startGame();
        }
```